### DISCUSSION TITLE
Can I skip opening connections for certain requests?

### DISCUSSION THREAD

**Discussion by ar-qun** [pallets-eco/flask-sqlalchemy#1339] [Q&A] [answered]
Hello everyone,

I was wondering if it was possible to skip creating a session or connecting to the DB for certain paths?

For example, I have a /healthz for Kubernetes liveness check and it would make sense to not connect there because it is not being used. I am sure there are other use cases as well.

Thanks in advance!

**Comment by ar-qun** [comment]
My assumption being that it is connection on those requests.

**Comment by davidism** [comment, marked as answer]
If you don't access the connection, it's not created.
